apps/aot/src/app/api/uploadthing/route.ts

```diff
@@ -1,7 +1,7 @@
-import { createNextRouteHandler } from 'uploadthing/next';
+import { createRouteHandler } from 'uploadthing/next';
```

Comment:
Per https://docs.uploadthing.com/v7#deprecations-have-been-removed, `createNextRouteHandler` is now `createRouteHandler`.